src/main/kotlin/dynamic_programming/PalindromePartitioning.kt

```diff
@@ -0,0 +1,75 @@
+package dynamic_programming
+
+/**
+ * Palindrome Partitioning Algorithm
+ *
+ * You are given a string as input, and task is to find the minimum number of partitions to be made,
+ * in the string sot that the resulting strings are all palindrome
```

Comment:
Can you elaborate it with some examples?

Reply:
the suggested changes have been made, thank you for your insight